packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -82,13 +86,21 @@ const chatMessageStyles: ComponentSlotStylesInput<
     position: 'absolute',
     right: v.actionMenuPositionRight,
     top: v.actionMenuPositionTop,
-    overflow: p.focused ? 'visible' : 'hidden',
 
-    // hide and squash actions menu to prevent accidental hovers over its invisible area
-    opacity: p.focused ? 1 : 0,
-    width: p.focused ? 'auto' : 0,
-  }),
+    ...(_.isNil(v.showActionMenu) && {
+      overflow: p.focused ? 'visible' : 'hidden',
+      // hide and squash actions menu to prevent accidental hovers over its invisible area
+      opacity: p.focused ? 1 : 0,
+      width: p.focused ? 'auto' : 0,
+    }),
 
+    ...(v.showActionMenu !== undefined && {
+      visibility: 'hidden',
```

Comment:
I'm affraid this will break accessibility for JAWS

Reply:
Will the combinatino of these props works better:
```
      overflow: v.showActionMenu ? 'visible' : 'hidden',
      // hide and squash actions menu to prevent accidental hovers over its invisible area
      opacity: p.showActionMenu ? 1 : 0,
      width: p.showActionMenu ? 'auto' : 0,
```

Reply:
I changed it to work with this three props, from visual point of view, it behaves the same. Let me know if this is ok